### ISSUE TITLE
What settings should I choose.

### ISSUE THREAD

**Issue by jeanlain** [cschin/peregrine-2021#17]
Hello, I have tested this tool to assemble a genome on sequel II data (ccs), but I have no idea what setting I should tweak to improve the results. The documentation doesn't really explain what the settings are.
For instance, the assembly size appears too long, like 60% longer than the expected genome size. I suspect this is because the reads come from a mixture of individuals (there isn't enough DNA in one individual par HiFi sequencing) and they are quite heterozygous (about 1 SNP every 100 bases). But I have no idea what setting I should tweak to avoid generating different contigs for different haplotypes (I don't care about phasing, I first want to generate a good assembly). 
Should I tweak the "alignment tolerance" argument? What does it do?

**Comment by cschin** [comment]
yes. for you case, you need to turn of the error corrections and allow fuzzy match to get longer contigs.  have you tried hifiasm? or hiCanu? how do the results look like there?
